.github/workflows/release-tag-versions.yaml

```diff
@@ -0,0 +1,113 @@
+name: Reusable workflow for releasing tag versions
+
+on:
+  workflow_call:
+    inputs:
+      version-configs-dir:
+        description: "The Path to the directory containing the file where the versioning is defined and `.bumpversion.cfg` file."
+        required: true
+        type: string
+      release-type:
+        description: "Scope of the release (major, minor or patch)."
+        required: true
+        type: string
+      next-dev-release-type:
+        description: "Scope of the next release (minor or patch) for developers"
+        required: true
+        type: string
+      next-dev-release-suffix:
+        description: "The suffix to add for the developer version"
+        required: false
+        default: "SNAPSHOT"
+        type: string
+    secrets:
+      github-email:
+        description: "The GitHub email for committing the changes."
+        required: true
+      github-username:
+        description: "The GitHub username for committing the changes."
+        required: true
+      github-token:
+        description: "The GitHub token for committing the changes."
+        required: true
+
+jobs:
+  bump_and_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out default branch
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.github-token }}
```

Comment:
please set `persist-credentials: false`
otherwise pushing to protected branches is not possible
for reference https://github.com/bakdata/ci-templates/tree/main/actions/commit-and-push#commit-and-push